canonical-websites/www.ubuntu.com
# One redirect for all USN URL versions

Redirect all versions of URN URLs to the correct place:

- /usn-NUMBER
- /usn/usnNUMBER
- /usn/usn-NUMBER
- /usn/USN-NUMBER

This hopefully improves upon and replaces https://github.com/canonical-websites/www.ubuntu.com/pull/2850

Fixes canonical-websites/usn.ubuntu.com#37

QA
--

``` bash
./run
```

Then in another window, check all USN URL forms redirect to the correct URL directly:

``` bash
$ curl -sI 127.0.0.1:8001/usn/USN-3593-1/ | grep Location
Location: https://usn.ubuntu.com/3593-1/
$ curl -sI 127.0.0.1:8001/usn/usn-3593-1/ | grep Location
Location: https://usn.ubuntu.com/3593-1/
$ curl -sI 127.0.0.1:8001/usn-3593-1/ | grep Location
Location: https://usn.ubuntu.com/3593-1/
$ curl -sI 127.0.0.1:8001/USN-3593-1/ | grep Location
Location: https://usn.ubuntu.com/3593-1/
$ curl -sI https://usn.ubuntu.com/3593-1/ | grep HTTP
HTTP/1.1 200 OK
```
